components/case_macros/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "case_macros"
+version = "0.1.0"
+edition = "2018"
+
+[lib]
+proc-macro = true
+
+[dependencies]
+quote = "1"
+syn = { version = "1", features = ["full", "extra-traits"] }
```

Comment:
Remove them, those two packages aren't actually needed.

Reply:
Got it